src/Sylius/Behat/Service/Mocker/PaypalApiMockerInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service\Mocker;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+interface PaypalApiMockerInterface
```

Comment:
Do you have proposal for implementation without ``MockerContainer``? I am not sure that naive implementation will cover enough this feature.